examples/basics/scene/mesh_shading.py

```diff
@@ -1,3 +1,13 @@
+# -*- coding: utf-8 -*-
+# vispy: gallery 30
+# -----------------------------------------------------------------------------
+# Copyright (c) Vispy Development Team. All Rights Reserved.
+# Distributed under the (new) BSD License. See LICENSE.txt for more info.
+# -----------------------------------------------------------------------------
+"""Show how to use mesh filters for shading (lighting) a mesh and displaying a
+wireframe.
+"""
```

Comment:
```suggestion
"""Show mesh filter usage for shading (lighting) a mesh and displaying a wireframe."""
```

I think we have the docstring tests turned off (or maybe not checking examples), but I think module docstrings should be one line.